tests/mbt/model/ccv.qnt

```diff
@@ -607,11 +793,11 @@ module ccv {
                 }
 
                 // check whether the validator has positive power
-                pure val provValSet = currentState.providerState.chainState.currentValidatorSet
+                pure val provValSet = currentState.providerState.chainState.currentValidatorPowers
                 pure val provValPower = if (provValSet.keys().contains(providerNode)) provValSet.get(providerNode) else 0
                 pure val consumersWithAddrAssignmentChangesInThisBlock = 
-                    if (provValPower > 0) {
-                        // if the consumer has positive power, the relevant key assignment for the consumer changed
+                    if (provValPower > 0 and providerNode.in(nonJailedNodes(currentState.providerState))) {
```

Comment:
Why don't we set the voting power of jailed validators to `0`?

Reply:
Because this makes unjailing operations hard, which we might want to add later. I filter the jailed validators out when the valset is entered on-chain during EndBlock